image_width: 744
image_height: 480
camera_name: stereo
camera_matrix:
  rows: 3
  cols: 3
  data: [416.8420112969208, 0, 368.4702740044705, 0, 417.406022956741, 232.4415236529705, 0, 0, 1]
distortion_model: plumb_bob
distortion_coefficients:
  rows: 1
  cols: 5
  data: [-0.251463327566586, 0.05129786869771439, -0.001425470575084714, -0.01127060661671339, 0]
rectification_matrix:
  rows: 3
  cols: 3
  data: [0.9026854089348176, -0.05311444365037127, 0.4270104312214077, 0.0308376494816883, 0.9977861883908282, 0.05892165672269335, -0.4291947015873163, -0.04001972179081106, 0.9023249580927929]
projection_matrix:
  rows: 3
  cols: 4
  data: [468.7020407619035, 0, -22.41032886505127, 126.8747297746739, 0, 468.7020407619035, 238.5082721710205, 0, 0, 0, 1, 0]